### ISSUE TITLE
Incompatible with Sniper mod

### ISSUE THREAD

**Issue by faithfulzeal** [MapleWheels/RoR2_QoLChanges#2]
Title

**Comment by MapleWheels** [comment]
Would kindly submit your `LogOutput` log file from BepInEx for this bug? And in what way is it incompatible, in game bugs or just doesn't launch? If in-game, please describe it and does this exist in both multiplayer and singleplayer or just one of the two?

**Comment by faithfulzeal** [comment]
In-game bugs, exists on single and multiplayer, it stops sniper from shooting at all.

**Comment by LemonToast-Chan** [comment]
I just discovered this issue myself. Here is the log. Hopefully this helps.
[LogOutput.log](https://github.com/MapleWheels/RoR2_QoLChanges/files/5249332/LogOutput.log)
If we're talking about specific issues, Sniper's shooting mechanic (left click, scoped shot, everything associated with shooting the gun) is disabled as well as skins not applying and forcing the default selection. In tandem with this issue, the framerate dips hard when using them. The backflip mechanic does nothing as well with both mods enabled. I have already made quarantined tests with both mods enabled, and can confirm they are incompatible.
